apps/04TowersOfHanoi.js

```diff
@@ -19,28 +19,53 @@ function printStacks() {
   console.log("c: " + stacks.c);
 }
 
-function movePiece() {
+function movePiece(startStack, endStack) {
   // Your code here
 
+  stacks[endStack].push(stacks[startStack].pop());
+
+
 }
 
-function isLegal() {
+function isLegal(startStack, endStack) {
   // Your code here
+  var lastItemFrom = stacks[startStack][stacks[startStack].length - 1];
+  var lastItemTo = stacks[endStack][stacks[endStack].length - 1];
+
+  // console.log(lastItemFrom);
+  // console.log(lastItemTo);
+  if (lastItemFrom < lastItemTo || lastItemTo === undefined) {
```

Comment:
I see you check for undefined for `lastItemTo`, but what about `lastItemFrom`? Is that possible? If so, what would that mean?